scss/application.scss

```diff
@@ -2,3 +2,26 @@
 @import 'variables/_colors';
 @import 'variables/_fonts';
 @import 'variables/_sizes';
+
```

Comment:
For styles, you should create a `styles.scss` and import them from `application.scss`.
The `application.scss` works as an organizer of all the styles but the styles are not here.